packages/ember-auto-import/ts/splitter.ts

```diff
@@ -252,13 +252,36 @@ export default class Splitter {
         target.importedBy,
         (imp) => imp.isDynamic
       );
+
       if (staticUses.length > 0) {
-        let bundleName = this.chooseBundle(staticUses);
-        deps.get(bundleName)!.staticImports.push(target);
+        let bundleNames = this.chooseBundle(staticUses);
+        bundleNames.forEach((name) => {
+          if (!deps.has(name)) {
+            deps.set(name, {
+              staticImports: [],
+              staticTemplateImports: [],
+              dynamicImports: [],
+              dynamicTemplateImports: [],
+            });
+          }
```

Comment:
Maybe extract this to a smaller helper function?

I'm thinking it might look like:

```js
bundleNames.forEach((name) => {
  const dep = ensureDep(deps, name);
  dep.staticImports.push(target); 
});
```

Reply:
Added this helper function